reco_utils/evaluation/python_evaluation.py

```diff
@@ -357,7 +357,7 @@ def merge_ranking_true_pred(
         col_item (str): column name for item
         col_rating (str): column name for rating
         col_prediction (str): column name for prediction
-        relevancy_method (str): method for determining relevancy ['top_k', 'by_threshold']
+        relevancy_method (str): method for determining relevancy ['top_k', 'by_threshold', None]
```

Comment:
What's the effect of setting `None`? Can we describe more?

Reply:
added comments